src/instructlab/log.py

```diff
@@ -67,7 +70,14 @@ def configure_logging(
         handler.close()
 
     stream = logging.StreamHandler()
-    stream.setFormatter(CustomFormatter(fmt=fmt))
+    try:
+        stream.setFormatter(CustomFormatter(fmt=fmt))
+    except ValueError as e:
+        raise ValueError(
+            f"Failed to configure log format: {e}\n"
+            "Have you specified a valid log format?\n"
+            "Consider reading: https://docs.python.org/3/library/logging.html#logrecord-attributes"
+        ) from e
```

Comment:
This validation is way too deep in the stack. You should validate the parameter in click or in the config loader instead.

```python
>>> import logging
>>> logging.PercentStyle("invalid").validate()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.11/logging/__init__.py", line 438, in validate
    raise ValueError("Invalid format '%s' for '%s' style" % (self._fmt, self.default_format[0]))
ValueError: Invalid format 'invalid' for '%' style
>>> logging.PercentStyle("%(message)s").validate()
```